react-web/src/components/JoinSession.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { UsernameForm } from './UsernameForm';
+
+export class JoinSession extends React.Component {
```

Comment:
If there's no further logic involved here, i think you can make this as a functional component instead.

Reply:
Cool, thanks for pointing this out. I hadn't seen this pattern